case_lowerb.c

```diff
@@ -1,8 +1,6 @@
 #include "case.h"
 
-void case_lowerb(s,len)
-char *s;
-unsigned int len;
+void case_lowerb(char *s, unsigned int len)
```

Comment:
`unsigned int len` feels like a `size_t`, let's see if the `int` -> `size_t` commit id up or downstream...

Reply:
It looks like there are multiple commits that convert `int` to `size_t` at multiple places. If this is something we want, would another branch be more appropriate or do we adjust source as we touch it?

Reply:
Yes, I would like to see it in a different PR on top of this, best once this is merged to avoid duplicate work.